modules/ecs/main.tf

```diff
@@ -3,7 +3,7 @@ locals {
 
```

Comment:
IIRC before upstreaming changes to these components, we test them out first in a client.

Has this already been tested in a client before merging here?